source/faq.txt

```diff
@@ -51,6 +51,15 @@ data in the operating system's credentials API. For more information on
 how Keytar operates and the specific APIs it accesses, refer to the
 `Keytar Github documentation <https://github.com/atom/node-keytar>`_.
 
+.. _compass-faq-logs:
+
+Where can I find Compass logs?
```

Comment:
[comment]

I think this section could benefit from being more direct. The first two sentences describe what logs *are*, but not where to find them. The final sentence reads more like a "see also" in its current form than a clear imperative. 

I recommend making the link language more explicit.